src/components/LandingPage/LandingPage.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { TEXT } from '@statics';
+import "./LandingPage.css"
+import Card from "@components/Card";
+
+const LandingPage = () => {
+    return (
+      <div>
+        <div id="landing-page-container">
+            <div id="info-section">
+                <div id="title">
+                    {TEXT.LANDING_PAGE.TITLE}
+                </div>
+                <div id="lab-description">
+                    {TEXT.LANDING_PAGE.DESCRIPTION}
+                </div>
+                <div className="btn_ubc_home">
+                <a href="https://www.ubc.ca" style={{display: "flex", justifyContent: "center", alignItems: "center", color: "#ffffff"}}>
+                    <p>{TEXT.LANDING_PAGE.UBC_PAGE_BUTTON}</p>
+                    </a>
+                </div>
```

Comment:
Perhaps tabbing here would be great for code style consistency :) 

```suggestion
                <div className="btn_ubc_home">
                  <a href="https://www.ubc.ca" style={{display: "flex", justifyContent: "center", alignItems: "center", color: "#ffffff"}}>
                      <p>{TEXT.LANDING_PAGE.UBC_PAGE_BUTTON}</p>
                      </a>
                </div>
```